vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -178,7 +178,9 @@ $global:map = @{
     );
     "c:\akse-cache\credential-provider\"    = @(
         "https://acs-mirror.azureedge.net/cloud-provider-azure/v1.29.2/binaries/azure-acr-credential-provider-windows-amd64-v1.29.2.tar.gz",
-        "https://acs-mirror.azureedge.net/cloud-provider-azure/v1.30.0/binaries/azure-acr-credential-provider-windows-amd64-v1.30.0.tar.gz"
```

Comment:
update v1.29/v1.30 to their latest patch version?

Reply:
I think we should have lots of dependency and CVE fixes in newer patch versions

Reply:
Done